includes/internal/emails/class-email-list-table.php

```diff
@@ -8,6 +8,8 @@
 namespace Sensei\Internal\Emails;
 
 use Sensei_List_Table;
+use Sensei_Pro_Upsell;
+use function add_query_arg;
```

Comment:
Is this `use` needed? I'm guessing it should continue working without it.

Reply:
Removed